orchestration/templates/cron-workflow.yaml

```
@@ -64,5 +66,5 @@ spec:
             - -H
             - 'Content-type: application/json'
             - -d
-            - '{"text": "Workflow $(WORKFLOW_ID) entered state: $(WORKFLOW_STATE)"}'
+            - '{"text": "Workflow $(WORKFLOW_ID) in env *$(WORKFLOW_ENV)* entered state: $(WORKFLOW_STATE)"}'
```

🤔 Might be good to *bold* the WORKFLOW_STATE as well, for readability?
```suggestion
            - '{"text": "Workflow $(WORKFLOW_ID) in env *$(WORKFLOW_ENV)* entered state: *$(WORKFLOW_STATE)*"}'
```